actions/xpubs/get.go

```diff
@@ -47,3 +47,42 @@ func (a *Action) get(c *gin.Context) {
 	contract := mappings.MapToXpubContract(xPub)
 	c.JSON(http.StatusOK, contract)
 }
```

Comment:
You should mark xpub endpoints as as deprecated, and maybe put in description, that user endpoint should be used instead.

It seams that `@Deprecated` comment should do the work

Reply:
use @DeprecatedRouter